.github/workflows/integration-tests.yml

```diff
@@ -30,34 +30,8 @@ jobs:
           distribution: 'temurin'
           cache: 'gradle'
 
-      - name: Gradle build service
-        run: ./gradlew --build-cache :service:build -x test
-
-  jib:
-    needs: [ build ]
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v3
-      - name: Set up JDK
-        uses: actions/setup-java@v3
-        with:
-          java-version: '17'
-          distribution: 'temurin'
-          cache: 'gradle'
-
-      - name: Construct docker image name and tag
-        id: image-name
-        run: |
-          GITHUB_REPO=$(basename ${{ github.repository }})
-          GIT_SHORT_HASH=$(git rev-parse --short HEAD)
-          echo "name=${GITHUB_REPO}:${GIT_SHORT_HASH}" >> $GITHUB_OUTPUT
-
-      - name: Build image locally with jib
-        run: |
-          ./gradlew --build-cache :service:jibDockerBuild \
-            --image=${{ steps.image-name.outputs.name }} \
-            -Djib.console=plain
+      - name: Build integration subproject
+        run: ./gradlew --build-cache :integration:build
```

Comment:
Before this was building the `service` subproject; this is incorrect as the `integration` subproject doesn't depend on `service` so it never needs to be built for this workflow. The build step could be removed completely, since `gradle runTest` will build it, although leaving it separate lets us split out build errors from integration tests errors, and keep the build time out of the test time.